nemo/collections/asr/models/ctc_models.py

```diff
@@ -99,8 +99,26 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         super().__init__(cfg=cfg, trainer=trainer)
         self.preprocessor = EncDecCTCModel.from_config_dict(self._cfg.preprocessor)
         self.encoder = EncDecCTCModel.from_config_dict(self._cfg.encoder)
+
+        if "params" in self._cfg.decoder:
+            if not self._cfg.decoder.params.feat_in and hasattr(self.encoder, 'feat_out'):
+                self._cfg.decoder.params.feat_in = self.encoder.feat_out
+            else:
+                logging.error("param feat_in of the decoder's config is not set!")
```

Comment:
Replace error message with a raised exception maybe? I dont remember if we handled this before.. Also error message should be `**.error("Parameter `feat_in` of the decoder's config is not set!")`

Reply:
done